### ISSUE TITLE
[React Native Gallery -> Permissions]: Announcing the Expand/Collapsed state for the ‘Request’ button which is incorrect.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#306]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Permissions’ button and invoke it.
Navigate to the ‘Request’ button and listen to the narrator announcement.

**Actual Result:**
Announcing the Expand/Collapsed state to the ‘Request’ button which is incorrect. When user navigate to the ‘request’ button, narrator is announcing as “Request windows.permission.appCaptureServices, button, collapsed”.

**Expected Result:**
Narrator should not announce the Expand/Collapsed state to the ‘Request’ button as it is not a Expand/Collapsed button.

**User Impact:**
Users with visual impairment who rely on screen reader will get confused if narrator gives incorrect information to the user.

Internal bug link: https://dev.azure.com/microsoft/OS/_workitems/edit/43361750

**Comment by YajurG** [comment]
I followed the repro steps for this issue and when navigating to any of the 'Request' buttons, Narrator would not announce 'Expand' or 'Collapsed'. It would simply say 'Request windows.permission.accessoryManager, button' or whichever permission was listed. I also looked at the UIA tree on Accessibility Insights and there was nothing regarding 'Expand' or 'Collapsed' state

**Comment by chrisglein** [comment]
@YajurG Is this related to https://github.com/microsoft/react-native-windows/issues/11734? Will it be fixed by the same PR?

**Comment by YajurG** [comment]
Yes it is related. I had to backport the change in [#11756](https://github.com/microsoft/react-native-windows/pull/11756) to 0.71 and then test the change in Gallery, which is working now. Pushing the PR which will close this issue.
